modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java

```diff
@@ -2402,6 +2395,19 @@ else if (!_isClientErrorException(graphQLError)) {
 			return processedErrors;
 		}
 
+		private Throwable _getCauseThrowable(GraphQLError graphQLError) {
```

Comment:
I would rename as `_getThrowable` only because you are getting a Throwable instance, not a `CauseThrowable` instance

Reply:
Ok! I will do it